test/integration/requests/pathFind.ts

```diff
@@ -45,4 +52,57 @@ describe('path_find', function () {
 
     assert.deepEqual(response, expectedResponse)
   })
+
+  // For other stream style tests look at integration/requests/subscribe.ts
+  it('path_find stream succeeds', async function () {
+    const wallet2 = await generateFundedWallet(this.client)
+    const pathFind: PathFindRequest = {
+      command: 'path_find',
+      subcommand: 'create',
+      source_account: this.wallet.getClassicAddress(),
+      destination_account: wallet2.getClassicAddress(),
+      destination_amount: '100',
+    }
+
+    const response = await this.client.request(pathFind)
+
+    const expectedResponse: PathFindResponse = {
+      id: response.id,
+      status: 'success',
+      type: 'response',
+      result: {
+        alternatives: response.result.alternatives,
+        destination_account: pathFind.destination_account,
+        destination_amount: pathFind.destination_amount,
+        source_account: pathFind.source_account,
+        full_reply: false,
+        id: response.id,
+      },
+    }
+
+    const expectedStreamResult: PathFindStream = {
+      type: 'path_find',
+      source_account: pathFind.source_account,
+      destination_account: pathFind.destination_account,
+      destination_amount: pathFind.destination_amount,
+      full_reply: true,
+      id: 10,
+      alternatives: [],
+    }
+
+    const event = new Promise((resolve, reject) => {
+      const client: Client = this.client
+      client.on('path_find', (path) => {
+        assert.equal(path.type, 'path_find')
+        assert.deepEqual(_.omit(path, 'id'), _.omit(expectedStreamResult, 'id'))
+        resolve('success')
+      })
+    })
```

Comment:
The better way to handle this is to use `done` instead of this promise structure. See some of the tests in `test/connection.ts`.

Reply:
The reason that this function can't use the `done` structure is because I was using `await` in order to get access to the ledger's response, and I can't use a callback and await in the same test (It gives an error that the resolution is ambiguous because await makes the function async, which forces it to return a promise and mocha can't tell if it's supposed to finish when the promise resolves or when the callback happens). 

So to avoid that issue I wrapped the callback into a promise to clarify that the test ends when that promise returns.

Maybe the underlying problem is that I don't need to use `await`s?